### PR TITLE
Uninstall extension warning modal

### PR DESCRIPTION
**Pull request by alicjakujawa**
## Description

This PR adds in warning modal with confirmation input whenever user want to uninstall extension

**New stuff** ✨

* ExtensionUninstallConfirmDialog
* extension message descriptors
* Created map with content for dialog depend on extension id

**Changes** 🏗

* Displaying ExtensionUninstallConfirmDialog with "I UNDERSTAND" input whenever user want to uninstall extension

Voting reputation warning modal:
WARNING

Default & Payment extension modal
Screenshot 2021-05-18 at 12 41 42

Resolves DEV-347

### CODE REVIEW

**Review by chinins — APPROVED** [review]
Looks good!

File: src/modules/dashboard/components/Extensions/ExtensionUninstallConfirmDialog/ExtensionUninstallConfirmDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  onClick = () => close(null),
+}: Props) => {
+  const { formatMessage } = useIntl();
+  const [isWarningInputValid, setIsWarningInputValid] = useState<boolean>(
+    false,
+  );
+
+  const onWarningInputChange = (e) => {
+    setIsWarningInputValid(e.target.value === 'I UNDERSTAND');
+  };
```
Comment (rdig) [inline src/modules/dashboard/components/Extensions/ExtensionUninstallConfirmDialog/ExtensionUninstallConfirmDialog.tsx:35]: I would argue that this should be a single word, not two.

So I would vote for something like "UNINSTALL". That way you can just copy/paste to uninstall it :)

**Review by rdig — APPROVED** [review]
Very nice work @alicjakujawa :100:

File: src/modules/dashboard/components/Extensions/ExtensionUninstallConfirmDialog/ExtensionUninstallConfirmDialog.tsx (hunk 1 above)
Comment (alicjakujawa) [reply src/modules/dashboard/components/Extensions/ExtensionUninstallConfirmDialog/ExtensionUninstallConfirmDialog.tsx:35]: So you need to argue with product team :D

**Pull request merged**